actionmailer-balancer.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |spec|
   spec.metadata['changelog_uri'] = 'https://github.com/railsware/actionmailer-balancer/blob/main/CHANGELOG.md'
   spec.metadata['rubygems_mfa_required'] = 'true'
 
-  spec.add_runtime_dependency 'actionmailer', '> 4.0', '< 8.0'
+  spec.add_dependency 'actionmailer', '> 4.0', '< 8.0'
```

Comment:
do you know why it was `add_runtime_dependency` before?

Reply:
`add_runtime_dependency` is the same thing. apparently back then there were no such rule in Rubocop probably.